seed/views/organizations.py

```diff
@@ -205,7 +203,6 @@ class OrganizationUsersSerializer(serializers.Serializer):
 
 class OrganizationViewSet(viewsets.ViewSet):
     raise_exception = True
-    authentication_classes = (SessionAuthentication, SEEDAuthentication)
```

Comment:
seems like we should get rid of all of these (e.g. https://github.com/SEED-platform/seed/blob/a27db02f2f837ffa3592f2baf5474b73ec4e6584/seed/api/v2/views.py#L13).

This will allow for the new oauth2 to work as expected because of this:
https://github.com/SEED-platform/seed/blob/986e7bdf5adc81477a1bd14683a3f9d12bb242c1/config/settings/common.py#L266

Reply:
agreed.  I thought I caught them all already, but I guess a couple slipped past me.